packages/cli/src/spec/inputHelpersSpec.ts

```diff
@@ -61,4 +61,17 @@ describe('inputHelpers', () => {
           inputHelpers.validateColor('rgb(23, 0 30')).toBeRejectedWithError();
     });
   });
+
+  describe('#validateDisplayMode', () => {
+    it('returns true for valid colors', async () => {
+      expect(await inputHelpers.validateDisplayMode('browser'));
+      expect(await inputHelpers.validateDisplayMode('minimal-ui'));
+      expect(await inputHelpers.validateDisplayMode('standalone'));
+      expect(await inputHelpers.validateDisplayMode('fullscreen'));
+    });
+
+    it('throws Error for invalid display modes', async () => {
+      await expectAsync(inputHelpers.validateColor('bogus')).toBeRejectedWithError();
```

Comment:
probably work adding a validation for an empty string too

Reply:
This function is now removed, but I added checking the empty string to the new `asDisplayMode` tests.